Automattic/wp-calypso
# Onboarding: remove /new (Gutenboarding & AnchorFM) flows

## Proposed Changes

This PR cleans up old code around Gutenboarding and AnchorFM flows.

## Testing Instructions

Smoke test the following flows:

* General onboarding (`/setup`)
* VideoPress (`/setup/videopress`), verify we can select both videomaker and the white variation (and actually applied to the site)
* Sensei (`/setup/sensei`), verify that we can finish the flow
* DIFM (`/start/do-it-for-me`), verify that the design picker still loads fine
   - I was not able to verify that the selected design is correct, even in production. Because the created site seems to use the default theme (TT4) no matter what theme I selected in the design picker step... any input is welcome. Maybe we only activate the design manually based on the content request.
   - I've at least verified that the `calypso_signup_select_design` event is fired correctly.

## Pre-merge Checklist
